app/templates/cookies/cookie_settings.html

```diff
@@ -21,12 +21,18 @@
 
 {% block mainContent %}
 
-  {{ dmAlert({
-    "type": "error",
-    "classes": "dm-cookie-settings__error",
+  {{ govukErrorSummary({
     "titleText": "There was a problem saving your settings",
-    "text": "Please select 'Yes' or 'No'.",
-    "attributes": [("id", "dm-cookie-settings-error")]
+    "classes": "dm-cookie-settings__error",
```

Comment:
We should either remove this class or get rid of the @include in its scss (which is using toolkit styles which don't affect the Error Summary component): https://github.com/alphagov/digitalmarketplace-user-frontend/blob/65b0b340ec9d5554c9ed22ae988b5c56bece97af/app/assets/scss/overrides/_cookie-settings.scss#L24-L27

To remove it, looks like we'd have to call [this function](https://github.com/alphagov/digitalmarketplace-govuk-frontend/blob/eb4c7fc9e9d0732421cce8307d181aab6a740388/src/digitalmarketplace/components/cookie-settings/cookie-settings.js#L105-L111 ) on load, so maybe better to keep the display: none override.